features/profile/src/main/res/values-ro-rRO/strings.xml

```diff
@@ -8,6 +8,9 @@
     <string name="setting_notification">Notificări</string>
     <string name="setting_logout">Log out</string>
     <string name="setting_delete_account">Șterge cont</string>
-    <string name="label_change_picture">Vrei să îți schimbi fotografia?</string>
-    <string name="label_upload_picture">ÎNCARCĂ FOTOGRAFIE</string>
+    <string name="question_change_picture">Vrei să îți schimbi fotografia?</string>
+    <string name="label_upload_picture">Încarcă fotografie</string>
+    <string name="question_delete_account">Ești sigur că vrei să îți ștergi contul?</string>
+    <string name="warning_delete_account">Această acțiune este ireversibilă. Dacă îți ștergi contul vei pierde tot istoricul de lecții.</string>
+    <string name="label_delete_account">@string/setting_delete_account</string>
```

Comment:
Do we need to declare this string twice? Why not using setting_delete_account for both cases?

Reply:
I thought of them to be two string resources that by coincidence happened to have the same value. The setting_change_photo isn't used in the change photo dialog, why would setting_delete_account be? That was my quick reasoning. But I could change it to only one string if you think it's better.

Reply:
A better option doesn't come to my mind now, maybe we can change it in the future.